tests/test_fieldtypes.py

```diff
@@ -452,7 +453,7 @@ def test_datetime():
         ("2006-11-10T14:29:55.585192699999999-07:00", datetime(2006, 11, 10, 21, 29, 55, 585192, tzinfo=UTC)),
     ],
 )
-def test_datetime_formats(tmp_path, value, expected_dt):
+def test_datetime_formats(tmp_path, value, expected_dt) -> None:
```

Comment:
Could you also add types to the arguments?

Reply:
Added.